packages/react-core/src/layouts/Split/SplitItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/layouts/Split/styles.css';
+import { componentShape } from '../../internal/componentShape';
+import { css, getModifier } from '@patternfly/react-styles';
+
+export const SplitItemVariant = {
+  primary: 'primary',
+  secondary: 'secondary'
+};
+
+const propTypes = {
+  variant: PropTypes.oneOf(Object.keys(SplitItemVariant)),
+  children: PropTypes.node,
+  className: PropTypes.string,
+  component: componentShape
+};
+
+const defaultProps = {
+  variant: null,
+  className: '',
+  children: null,
+  component: 'div'
+};
+
+const SplitItem = ({ variant, className, children, ...props }) => (
+  <div
```

Comment:
use the component from props

Reply:
The component from props shouldn't even be there.